frontend/pages/queries/edit/components/EditQueryForm/EditQueryForm.tsx

```diff
@@ -735,6 +746,16 @@ const QueryForm = ({
                     label="Logging"
                     wrapperClassName={`${baseClass}__form-field ${baseClass}__form-field--logging`}
                   />
+                  {!isLoadingAppConfig && (
+                    <DiscardDataOption
+                      {...{
```

Comment:
Is this `...` necessary? Aren't these just props that can be set directly?

Reply:
Gotcha. Yeah, I'm a fan of the spread syntax generally and the verbosity of react's component prop syntax has always been a little annoying. In any case, I like how this example groups all the props into one place. See my comment below regarding the mixed pattern.